src/components/SavedFiltersMenu.vue

```diff
@@ -54,9 +66,33 @@
     },
   })
 
-  const canSave = computed(() => internalSavedSearch.value?.name === customSavedSearch.name && can.create.saved_search)
+  const isCustomUnsavedFilter = computed(() => internalSavedSearch.value?.name === customSavedSearch.name)
+  const canSave = computed(() => isCustomUnsavedFilter.value && can.create.saved_search)
   const canDelete = computed(() => internalSavedSearch.value?.id && can.delete.saved_search)
 
+  const canToggleDefault = computed(() => {
+    // can't set the default to an unsaved filter. save it first
```

Comment:
Technically there's no reason why we couldn't let users set a custom unsaved filter as their default right? 

I wonder if we should just let users set anything as their default. I imaging that would cause issues with the toast when filter was deleted. But maybe we're overthinking that? I liked it when I saw you demo it but I almost thing letting users save any filters as their default and not just saved filters might be better even if we lose the toast.

Reply:
Ah that's an interesting thought 🤔 . 

especially because I'm saving the entire _filter_ itself and not just the name (which at some point I thought to myself how much easier that might've been).

Reply:
Yeah the name would have been easier! But since you've got the whole filter, we could just use it right? And if it happens to match up with a saved filter we'll show that as the selected and default filter.